app/components/pages/SubmissionWizard/steps/Editors/FormSections.js

```diff
@@ -20,17 +19,17 @@ export const SuggestedReviewer = ({ changeHandler, index }) => (
     <ValidatedField
       label={
         index < limits.suggestedReviewers.min
-          ? 'Suggested reviewer name'
-          : 'Optional suggested reviewer name'
+          ? `Suggested reviewer ${index + 1} name`
+          : `Suggested reviewer ${index + 1} name (optional)`
```

Comment:
@diversemix was "suggested" left in here on purpose? The designs just say "Reviewer 1 name", "Reviewer 2 email", etc:
<img width="782" alt="screen shot 2018-08-21 at 10 02 34" src="https://user-images.githubusercontent.com/17023702/44392263-b4c74300-a529-11e8-8d65-7d0d26af11e7.png">

Reply:
Good catch ... I'd leave it for now and @hdrury1 can make a call once she is back.